examples/autodetect-istio/rendered_manifests/daemonset.yaml

```diff
@@ -30,10 +30,10 @@ spec:
         app: splunk-otel-collector
         component: otel-collector-agent
         release: default
+        sidecar.istio.io/inject: "false"
```

Comment:
Which istio version introduced this support? If it's recent, we might want to keep both for now. Otherwise, we need to update the docs/comments for supported istio versions, if we have any.

Reply:
From documentation, the [annotation has been deprecated](https://istio.io/v1.15/docs/reference/config/annotations) since [Istio v1.15](https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases), which has been EOL since April 2023.

The label with the same name has been in place since [Istio v1.16](https://istio.io/v1.16/docs/reference/config/labels/), and labeled `beta` stability since that release.